src/components/Hero.style.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const DivStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`;
+
+export const H1Styled = styled.h1`
+  font-family: Bitter;
```

Comment:
the font-styles like family, size, weight etc should be defined in global styles. They will be used throughout the app and are usually the same for all h1, all h2, etc